unit-tests-jest/numbers.test.js

```diff
@@ -0,0 +1,106 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers';
+
+describe('evenNumbers', () => {
+  it('returns the even numbers', () => {
+    const numbers = [1, 4, 5, 10, 0];
+    const result = evenNumbers(numbers);
+    expect(result).toStrictEqual([4, 10, 0]);
```

Comment:
toEqual works just as well in this case, unless you want to allow `undefined` as an array value.